changelog.d/pack-integration-tools.md

```
@@ -0,0 +1,8 @@
+---
+category: Added
+---
+
+- Added MCP tool to list integration packs and bundles installed in a Rewst organization.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)